Frontend/ui-driver/src/Resource/Localizable/Strings.purs

```diff
@@ -0,0 +1,48 @@
+{-
+ 
+  Copyright 2022-23, Juspay India Pvt Ltd
+ 
+  This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+ 
+  as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+ 
+  is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ 
+  or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+ 
+  the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module Language.Strings where
+
+import Data.String.Common (trim)
+import Language.Types (STR)
+import MerchantConfig.Utils (getStringFromConfig, getStringWithVar)
+import Prelude (($))
+import ConfigJBridge (getKeyInSharedPrefKeysConfig)
+import Data.Maybe (Maybe(..))
+import Resources.Localizable.BN (getBN)
+import Resources.Localizable.EN (getEN)
+import Resources.Localizable.HI (getHI)
+import Resources.Localizable.KN (getKN)
+import Resources.Localizable.ML (getML)
+import Resources.Localizable.TA (getTA)
+
+data Language = EN_US | KN_IN | HI_IN | TA_IN 
+
+getString :: STR -> String
```

Comment:
if this function/ file is same can it be under common library?

Reply:
`Language` has no dependency so removed and `getString` has some differences in both customer and driver so it is required to be seperate

Reply:
what's the difference in `getString` I tried finding out one but looks similar

Reply:
`getString` is using `STR` which is different for both, also it is calling `getStringFromLocal ` which requires language string which is also different for both

Reply:
these functions can be passed as params to getString from driver or customer app independently